scrapyd_k8s/config.py

```diff
@@ -21,13 +21,27 @@ def launcher_cls(self):
         pkg, cls = repo.rsplit('.', 1)
         return getattr(import_module(pkg), cls)
 
+    def joblogs(self):
+        if self._config.has_section('joblogs'):
+            return self._config['joblogs']
+        else:
+            return None
+
+    def joblogs_storage(self, provider):
+        if not self._config.has_section('joblogs.storage.%s' % provider):
+            return None
+        return self._config['joblogs.storage.%s' % provider]
+
```

Comment:
Would `self._config.get('joblogs.storage.%s' % provider)` be shorter and do the same? (also `joblogs`)

Reply:
Well, the signature of the get method says it is used to retrieve the value of a particular option, not the entire section and requires an option as a second argument, that is why I don't think it is suitable here, since it has a different logic
```
get(section, option, *, raw=False, vars=None, fallback=_UNSET)
```

Reply:
Ah, that makes sense. Sorry, I was just looking at what the code around did and didn't think about values/sections. Fine!